tests/testthat/test-progress.R

```diff
@@ -35,11 +35,13 @@ with_mock_HTTP({
                 content=readBin(url, "raw", 4096), ## Assumes mock is under 4K
                 status_code=200, headers=list(`Content-Type`="application/json")))
         },
-        test_that("Progress polling goes until 100", {
-            expect_output(
+        test_that("Progress polling goes until 100 and has a newline", {
+            out <- capture.output(
                 expect_equal(pollProgress("https://app.crunch.io/api/progress/", wait=.001),
-                    100),
-                "=| 100%", fixed=TRUE)
+                             100), cat('command on next line'))
+            expect_equal(length(out), 2)
+            expect_match(out[1], "=| 100%", fixed=TRUE)
+            expect_match(out[2], "command on next line", fixed=TRUE)
```

Comment:
This seems awkward--why change away from `expect_output`? If there's a necessary reason, would you mind adding a comment explaining?